lib/doodbalib/__init__.py

```diff
@@ -159,10 +159,6 @@ def addons_config(filtered=True, strict=False):
             continue
         repos.discard(CORE)
         # Other addons fall in between
-        if len(repos) != 1:
```

Comment:
The correct fix would be probably:

```py
if filtered and len(repos) != 1:
```

Because it will usually run with `filtered=True`, except when building.